src/go/k8s/pkg/labels/labels_test.go

```diff
@@ -1,10 +1,11 @@
-package labels
+package labels_test
```

Comment:
let's make sure k8s is in the BSL license as a follow up change.

Reply:
Will do.